src/api/app/views/webui/request/show.html.erb

```diff
@@ -50,11 +50,16 @@
         <% end %>
         <% if @accept_at -%>
             <li><%= sprite_tag('exclamation', title: '') %>
-               <% if Time.now > Time.parse(@accept_at.to_s) %>
-                 This request will get accepted automatically when it enters the 'new' state!
+              <% if BsRequest::FINAL_REQUEST_STATES.include?(@state) %>
+                The request auto-accept was set to
```

Comment:
[Nitpick] 

Should be: "The request's auto-accept was set to"

Or @adrianschroeter s suggestion also works:
"Auto-accept was set to ..."